src/content/blog/announcements/2024-11-30_nixos-24.11.md

```diff
@@ -0,0 +1,52 @@
+---
+id: nixos-24.11
+title: NixOS 24.11 released
+date: 2024-11-30T11:00:00.000Z
+category: announcements
+---
+
+[![NixOS 24.11 Vicuña logo](../../../assets/logo/nixos-logo-24.11-vicuna-lores.png)](https://github.com/NixOS/nixos-artwork/blob/master/releases/24.05-vicuna/vicuna.png)
+
+Hello everyone, we are [Tristan Ross](https://github.com/RossComputerguy) and [Weijia Wang](https://github.com/wegank), the release managers for the newest release of NixOS known as NixOS 24.11 Vicuña. We're glad to announce that after the additional week-long delay that we can deliver this stable release of NixOS.
+
+NixOS is a Linux distribution, whose set of packages can also be used on other Linux systems and macOS.
+
+This release will receive bugfixes and security updates for seven months (up until 2025-06-31).
+
+- [Release manual](/manual/nixos/stable/release-notes.html#sec-release-24.11)
+  - [Highlights](/manual/nixos/stable/release-notes.html#sec-release-24.11-highlights)
+  - [New Services](/manual/nixos/stable/release-notes.html#sec-release-24.11-new-services)
+  - [Backward Incompatibilities](/manual/nixos/stable/release-notes.html#sec-release-24.11-incompatibilities)
+  - [Other Notable Changes](/manual/nixos/stable/release-notes.html#sec-release-24.11-notable-changes)
+- [Upgrade Instructions](/manual/nixos/stable/index.html#sec-upgrading)
+- [ISO Image Downloads](/download)
+
+The 24.05 release was made possible due to the efforts of **2669** contributors, who authored **49079** commits since the previous release.
```

Comment:
You meant 24.11, right?

Are those stats up-to-date?

Reply:
> You meant 24.11, right?

Yes

> Are those stats up-to-date?

Yes, I generated them yesterday. You can see they came from the NixOS release management chat on Matrix.